cypress/support/generate.js

```diff
@@ -0,0 +1,12 @@
+const faker = require('faker');
+
+function generateUser() {
+  const random = Math.random().toString().slice(2, 6);
```

Comment:
Use more specific variables' names.
```suggestion
  const randomNumber = Math.random().toString().slice(2, 6);
```